do.py

```diff
@@ -36,6 +36,15 @@ def lint():
 
 
 def test():
+    pretest_args = [
```

Comment:
we shouldn't write another test and run them separately. 

the goal is to run some code automatically before starting all tests. This can be done using pytest fixtures.

https://github.com/pytest-dev/pytest/issues/3709#issuecomment-406953552